list.md

```diff
@@ -380,3 +384,6 @@ David Zellhöfer (SBB Berlin)
 
 Philipp Zumstein (UB Mannheim)
 [https://github.com/zuphilip](https://github.com/zuphilip)
+
+Martin Malmsten (National Library of Sweden)
```

Comment:
We try to keep the list of people alphabetically sorted by their name.

Reply:
And of course it is not necessary to translate your library name to English. It has a very nice name in your language.